bindings/python/iota_sdk/client/client.py

```diff
@@ -35,7 +35,8 @@ def __init__(
         user_agent=None,
         local_pow=None,
         fallback_to_local_pow=None,
-        pow_worker_count=None
+        pow_worker_count=None,
+        _client_handle=None
```

Comment:
Not sure if that's the proper pythonic way. 
I would remove the _

Reply:
How to add another constructor? I haven't really seen an example that didn't reuse the init one

Reply:
What if we make a special hidden function for passing an existing client?

Reply:
Yeah, true that...

So I guess just remove the underscore as it doesn't really help with anything and we are good 😆